src/core/SpriteAnimator.tsx

```diff
@@ -472,7 +571,7 @@ export const SpriteAnimator: React.FC<SpriteAnimatorProps> = /* @__PURE__ */ Rea
     })
 
     // utils
-    const getFirstItem = (param: any): any => {
+    const getFirstItem = (param: object | null | []): any => {
```

Comment:
why type-casting the returned value with `: any`?

Reply:
fixed on latest commit